joomla/joomla-cms
# In Joomla 4 - Rename Control Panel as Home Dashboard

Pull Request for Issue #29477 .

Summary of Changes

All of the Dashboards in Joomla 4 have xxx Dashboard in the Title bar, except the page linked to by the Home Dashboard menu item, which is entitled Control Panel. This pull request simply changes the page title from Control Panel to Home Dashboard. There are 11 occurrences of "Home Dashboard" and 8 occurrences of "Control Panel". After applying this pull request there is 1 occurrence of "control panel" which refers to the hosting control panel.

Testing Instructions

Select Home Dashboard from the Administrator menu.

Actual result BEFORE applying this Pull Request

The page title is Control Panel

Expected result AFTER applying this Pull Request

The page title is Home Dashboard

Documentation Changes Required

The Help screen changes are in hand - that is where this anomaly came to light. There are likely to be other mentions of Control Panel in Tutorials.
